addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3568,6 +3568,19 @@ options.registry.sizing.include({
             $body.on('mouseup', documentMouseUp);
         });
         return defs;
+    },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    async updateUIVisibility() {
+        await this._super(...arguments);
+        if (this.$target[0].classList.contains('s_table_of_content_vertical_navbar') ||
+                this.$target[0].classList.contains('s_table_of_content_horizontal_navbar')) {
```

Comment:
You could have simply checked the "s_table_of_content_navbar_wrap" class which is common to the 2 position options (vertical and horizontal). But I wonder if we shouldn't make a more generic solution for this instead... For instance, add "forceNoDragAndDropButton: true," in "options.registry.TableOfContentNavbar". That way, we don't have to add classes in this condition if we have to do the same thing for another snippet later, but just add the parameter. The same way we did for "forceNoDeleteButton".

Reply:
@bvr-odoo I now simply removed the move/drag options - leaving the Position option available to achieve the same result.

Reply:
@bso-odoo  & @bvr-odoo , I confirm I'm fine with the removing of 'drag & drop' on the TOC navbar.
My recommendation would have been;
Both for navbar & content, remove the drag & drop.
However, leave the arrows (left right/top bottom).
But Benoit's solution is ok as well :)